sklearn/cluster/tests/test_optics.py

```diff
@@ -245,6 +246,22 @@ def test_dbscan_optics_parity(eps, min_samples):
     assert percent_mismatch <= 0.05
 
 
+def test_min_samples_min_clusters_float():
+    # check if OPTICS is callable with float parameters for
+    # min_samples and min_cluster_size
+    C1 = [[0, 0], [0, 0.1], [0, -.1]]
+    C2 = [[10, 10], [10, 9], [10, 11]]
+    C3 = [[100, 100], [100, 96], [100, 106]]
+    X = np.vstack((C1, C2, C3))
+
+    expected_labels = np.r_[[0] * 3, [1] * 3, [2] * 3]
```

Comment:
If the point is to test the equivalence between float and int values, then you should be comparing the results of different OPTICS ruins to each other rather than compare to a ground truth again...

Reply:
I don't want to introduce a new test for 0.21.3, I think we can make use of existing tests.